connect-inject/handler.go

```diff
@@ -335,34 +335,15 @@ func (h *Handler) overwriteProbes(ns corev1.Namespace, pod *corev1.Pod) error {
 	}
 
 	if tproxyEnabled && overwriteProbes {
-		if cs := pod.Spec.Containers; len(cs) > 0 {
-			// The first container will be the original application container that we need to
-			// modify probes for.
-			appContainer := cs[0]
-			if appContainer.LivenessProbe != nil && appContainer.LivenessProbe.HTTPGet != nil {
-				// We need to save original port first so that endpoints controller can use it for exposing paths.
-				pod.Annotations[annotationOriginalLivenessProbePort] = appContainer.LivenessProbe.HTTPGet.Port.String()
-				listenerPort := defaultExposedPathsListenerPortLiveness
-				if raw, ok := pod.Annotations[annotationTransparentProxyLivenessListenerPort]; ok {
-					listenerPort, err = strconv.Atoi(raw)
-					if err != nil {
-						return err
-					}
-				}
-				appContainer.LivenessProbe.HTTPGet.Port = intstr.FromInt(listenerPort)
-
+		for i, container := range pod.Spec.Containers {
```

Comment:
should we skip envoy sidecar here just in case? Currently it doesn't have probes defined, but in case it does in the future, it'd be good to omit it just in case.

Reply:
done!!